pkg/apis/build/v1beta1/build_types.go

```diff
@@ -176,6 +176,34 @@ func (buildSpec *BuildSpec) StrategyName() string {
 	return buildSpec.Strategy.Name
 }
 
+// VulnerabilityIgnoreOptions refers to ignore options for vulnerability scan
+type VulnerabilityIgnoreOptions struct {
+
+	// Issues references the security issues to be ignored in vulnerability scan
+	Issues []string `json:"issues,omitempty"`
```

Comment:
@SaschaSchwarze0 isn't ID a better name. Just by looking at my Trivy output, they use VulnerabilityID. Issues does not fully correlate. opinions @karanibm6 ?

Reply:
@SaschaSchwarze0 @HeavyWombat can u pls provide an opinion on the above.

Reply:
I tend to agree. In the BuildRun status, we have ID and Severity. Let's use IDs here also.